doc/nrf-bm/app_dev/dfu/memory_partitioning.rst

```diff
@@ -103,48 +103,55 @@ The placement of the SoftDevice in memory is predetermined and must align with s
++--------------------------+---------------------+-----------------------+-----------------------+-------------------------+
+| ``slot1_partition``      | Firmware loader     | 64 kB                 | 48 kB                 | 32 kB                   |
++--------------------------+---------------------+-----------------------+-----------------------+-------------------------+
+| ``softdevice_partition`` | SoftDevice          | \-                    | \-                    | \-                      |
++--------------------------+---------------------+-----------------------+-----------------------+-------------------------+
+| ``metadata_partition``   | Stores metadata     | 0.5 kB                | \-                    | \-                      |
++--------------------------+---------------------+-----------------------+-----------------------+-------------------------+
 
 .. note::
    The sizes and configurations of slot0 and slot1 are asymmetrical.
 
-The boards in |BMshort| have a reserved size of 31 kB for MCUboot and 64 kB for FW loader (including MCUboot meta data).
-The sizes are set large enough to include logging and other debugging information required for development.
-If you want to keep optimization and log level equal to the default in the sample but want to maximize the size available for application and storage, you can adjust the partition as recommended in Development column.
-The Release column shows the recommended size to be used, given that the project is optimized as described.
-You cannot update the ``boot_partition`` after release.
-Set the size to match the size given by the project.
-Set the size for the ``slot1_partition`` to accommodate for updates.
-There is a recommended size for both partitions, but you need to consider how much space is needed for each partition in your project.
+* **DK board** - Represents the partition size that you can find in the board definition files (.dts) that have been defined for the MCUboot board target in |BMshort|.
+  These sizes are not optimized but set large enough to provide enough memory to do development on multiple domains without having to adjust the partition size.
 
-If you are not using a DK board target, refer to :ref:`ug_dfu_preparing_dfu_board` for how to partition the memory using the devicetree.
+* **Development** - Represents the partition size that can be used to maximize the application space when running with default configuration of the MCUboot and firmware loader project.
+  This setting does not include extra space in the ``slot1_partition`` to accommodate for updates without the risk of having to adjust the partition size.
+
+* **Release** - Represents the partition size recommended when the MCUboot and firmware loader projects are optimized for release.
+  See :ref:`dfu_memory_partitioning_optimize` for details.
+  The ``boot_partition`` (MCUboot) cannot be updated after the release and the size must be set to match the project.
```

Comment:
The "must" seems a bit to strong/ restrictive. I would use "..... should be set to match the project"